frontend/src/ui/Blocks/Series.tsx

```diff
@@ -166,24 +168,29 @@ const ReadySeriesBlock: React.FC<ReadyProps> = ({
     const timeMs = (event: Event) =>
         new Date(event.syncedData?.startTime ?? event.created).getTime();
 
+    const sortEvents = (a: Event, b: Event) =>
```

Comment:
I don't think `sort` is the correct term here. This is a comparator, so maybe `eventComparator` or `compareEvents`?